src/styles/stopWatch.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  width: 100vw;
+  height: 100vh;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopWatch {
+  position: relative;
+
+  width: 50%;
+  height: 50%;
+
+  &__secondHand {
+    position: absolute;
+
+    height: 400px;
+    border-radius: 1px;
+    border: 8px solid blue;
+
+    top: 50%;
+    left: 50%;
+
+    animation: rotation 60s infinite steps(60);
+    transform-origin: center top;
+  }
+
+  &__minuteHand {
+    position: absolute;
+
+    height: 280px;
```

Comment:
And here, to avoid "going out" of the container without having fun with pixels :)